lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -1000,6 +845,104 @@ sub _resolve_workflow_for_build {
                 );
             }
         }
+        #Add gene category annotations to some output files from steps above. (e.g. determine which SNV affected genes are kinases, ion channels, etc.)
+        #AnnotateGenesByCategory - gene_category_<exome|wgs|wgs_exome>_<snv|indel>_result
+        my $annotate_genes_by_category_op = $self->annotate_genes_by_category_op($workflow);
+        $workflow->create_link(
+            source               => $converge_snv_indel_report_ops[-1],
```

Comment:
I like this idea. The qualities are actually a set of the mapping quality (mq) and the base quality (bq). So I'm not sure if just looking at the mq is sufficient. @gatoravi what would be the best way to determine the "best" set of mapping and base quality in the sireport_min_mq_bq list? Is looking at just the mqs and picking the highest sufficient?